### DISCUSSION TITLE
Stats for root domain redirect

### DISCUSSION THREAD

**Discussion by veerjd** [thedevs-network/kutt#456] [Ideas]
It'd be awesome to get the same stats feature for the root domain redirect

**Comment by veerjd** [comment]
Counts, etc.

**Comment by poeti8** [comment]
We already show stats, it's been literally advertised as the first thing on the homepage:

<img width="762" alt="image" src="https://user-images.githubusercontent.com/23660003/112977043-d6b9e500-916a-11eb-9d1d-1abcee9ad5b5.png">

**Comment by veerjd** [comment]
Can't see it for my custom url then

**Comment by veerjd** [comment]
I'm using the root url

**Comment by poeti8** [comment]
It has to have a visit. Sometime the count takes time to be updated if server is busy.

**Comment by veerjd** [comment]
It's been live for a few days and has been been clicked many times

**Comment by poeti8** [comment]
can you send a screenshot from the links table

**Comment by veerjd** [comment]
Don't see anything like it since I'm only using the root to redirect!

[Screenshot_20210330-115805.png](https://user-images.githubusercontent.com/26885867/113019233-5cc83280-914f-11eb-936f-0939e68c3c37.png)

**Reply by poeti8** [reply]
I said "links" table

**Reply by veerjd** [reply]
I do not see a "links" table:

![image](https://user-images.githubusercontent.com/26885867/113611205-348f7680-961c-11eb-9870-fec9cb272247.png)

**Reply by poeti8** [reply]
did you check other pages

**Reply by veerjd** [reply]
I'm sorry, I think you meant this. This is root url:

![image](https://user-images.githubusercontent.com/26885867/113612109-6523e000-961d-11eb-93df-988c0db2a413.png)

**Reply by poeti8** [reply]
There's no link. You have not shortened any URLs? Then for which link you were checking the stats?

**Reply by veerjd** [reply]
As per what I shared earlier, I only needed the root url of my domain to redirect. So I used the Home Page to define what the root of the domain should direct to.

**Reply by poeti8** [reply]
wow, ok, I'm not sure why you would want to do that, but we don't support that for the moment.

**Reply by veerjd** [reply]
Edited the request then

Thanks for the followup!
